src/Wallabag/CoreBundle/Resources/translations/messages.fa.yml

```diff
@@ -255,82 +255,82 @@ howto:
         description: 'این پیوند را به نوار بوک‌مارک مرورگرتان بکشید:'
 
 quickstart:
-    # page_title: 'Quickstart'
-    # intro:
-    #     title: 'Welcome to wallabag!'
-    #     paragraph_1: "We'll accompany you to visit wallabag and show you some features which can interest you."
-    #     paragraph_2: 'Follow us!'
-    # configure:
-    #     title: 'Configure the application'
-    #     language: 'Change language and design'
-    #     rss: 'Enable RSS feeds'
-    #     tagging_rules: 'Write rules to automatically tag your articles'
-    # admin:
-    #     title: 'Administration'
-    #     description: 'As an administrator, you have privileges on wallabag. You can:'
-    #     new_user: 'Create a new user'
-    #     analytics: 'Configure analytics'
-    #     sharing: 'Enable some parameters about article sharing'
-    #     export: 'Configure export'
-    #     import: 'Configure import'
-    # first_steps:
-    #     title: 'First steps'
-    #     new_article: 'Save your first article'
-    #     unread_articles: 'And classify it!'
-    # migrate:
-    #     title: 'Migrate from an existing service'
-    #     description: "Are you using another service? We'll help you to retrieve your data on wallabag."
-    #     pocket: 'Migrate from Pocket'
-    #     wallabag_v1: 'Migrate from wallabag v1'
-    #     wallabag_v2: 'Migrate from wallabag v2'
-    # developer:
-    #     title: 'Developers'
-    #     create_application: 'Create your third application'
-    # docs:
-    #     title: 'Full documentation'
-    #     annotate: 'Annotate your article'
-    #     export: 'Convert your articles into ePUB or PDF'
-    #     search_filters: 'See how you can look for an article by using search engine and filters'
-    #     fetching_errors: 'What can I do if an article encounters errors during fetching?'
-    #     all_docs: 'And so many other articles!'
-    # support:
-    #     title: 'Support'
-    #     description: 'If you need some help, we are here for you.'
-    #     github: 'On GitHub'
-    #     email: 'By email'
-    #     gitter: 'On Gitter'
+    page_title: 'Quickstart'
+     intro:
```

Comment:
Oh I see. Maybe when I was manually deleting comments to make translations, I should have deleted the extra space after the '#' character.
Is it only the red part above that needs to be fixed, or is it everywhere in the file?